packages/wpcom.js/src/lib/util/send-request.js

```diff
@@ -43,7 +43,8 @@ export default function sendRequest( params, query, body, fn ) {
 		params.apiVersion = query.apiVersion;
 		debug( 'apiVersion: %o', params.apiVersion );
 		delete query.apiVersion;
-	} else {
+	} else if ( ! params.apiVersion ) {
```

Comment:
This is a really weird bug. At least, I don't know why the other code doesn't trigger it. If this line isn't added, my wpcom.req.get call added above will fail with a 404 because it goes to apiVersion v1.1, which is the default version in wpcomjs.

I don't know why, when running the request through the redux data layer, this issue does not happen. The request there seems to go out under v1.2 correctly.

Reply:
This is because `apiVersion` is supposed to be in the second parameter of `wpcom.req.get()`, otherwise it gets overridden:

https://github.com/Automattic/wp-calypso/blob/37c161b6d44040aef789e62f7f199d27c6824806/packages/wpcom.js/src/lib/util/send-request.js#L42-L48

Ideally, we should avoid changing this to prevent other places from starting to behave unexpectedly. It's not a great place to be, but historically, this hasn't been providing a great API...

So I'd refrain from this change and pass `{ apiVersion: '1.2' }` as a second argument to the `.get()` above.

Reply:
Done 👍